src/lib/crypto/aes/operations.ts

```diff
@@ -0,0 +1,82 @@
+import { webcrypto } from 'one-webcrypto'
+import keys from './keys'
+import utils from '../utils'
+import { DEFAULT_SYMM_ALG, DEFAULT_CTR_LEN } from '../constants'
+import { SymmKey, SymmKeyOpts, SymmAlg, CipherText, Msg } from '../types'
+
+export async function encryptBytes(
+  msg: Msg,
+  key: SymmKey | string,
+  opts?: Partial<SymmKeyOpts>
+): Promise<CipherText> {
+  const data = utils.normalizeUtf16ToBuf(msg)
+  const importedKey = typeof key === 'string' ? await keys.importKey(key, opts) : key
+  const alg = opts?.alg || DEFAULT_SYMM_ALG
+  const iv = opts?.iv || utils.randomBuf(16)
+  const cipherBuf = await webcrypto.subtle.encrypt(
+    {
+      name: alg,
+      // AES-CTR uses a counter, AES-GCM/AES-CBC use an initialization vector
+      iv: alg === SymmAlg.AES_CTR ? undefined : iv,
+      counter: alg === SymmAlg.AES_CTR ? new Uint8Array(iv) : undefined,
```

Comment:
We always want to use AES-GCM and don't want that to be configurable. That also means we should always have an IV (it should never be undefined).

Reply:
I think they implemented this switch for switching between CTR and GCM.
I think all I need to do here is update the DEFAULT_SYMM_ALG

Reply:
While i'm at it, cleaning up references to non-supported crypto

Reply:
done. Our aes operations will only support GCM now.